### PR TITLE
Fix errors in maintainer guide

### LINKED ISSUE #798: Improve the maintainer guide
**Tell us about your environment**

* **MarkBind Version:** 2.0.0

* Vue-Strap step 2: Change from `-m` to `-am` in order to add the file first before committing. (i.e. `$ git commit -am 'Update dist'`).
* MarkBind step 2: This step should have been done AFTER `npm version`, because otherwise the `generator` meta-tag version number wouldn't have updated.
* MarkBind before step 4: Missing a `git commit --amend` to add in the new changes in the `generator` meta-tag (done in our current step 2) and `src/lib/markbind/package.json` version changes (done in our current step 3).
* MarkBind before step 4: Missing a `git tag delete vX.Y.Z` and `git tag add vX.Y.Z` to re-tag the newly amended version commit.
* Missing steps for closing issue milestones for the released version, and creating an issue milestone for the next version. 
* Missing section for new maintainers. Firstly, they need to ensure that they have admin rights to the GitHub repository and npm. Secondly, they need to call `npm adduser` to login to their account before they can publish a new version.

### PR DESCRIPTION
**Pull request by yamgent**
**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**

• [X] Documentation update

Fixes #798.

**What is the rationale for this request?**

There were some incorrect/missing instructions in the maintainer guide. See commit messages in each commit for details.

### CODE REVIEW

File: docs/devGuide/maintainerGuide.md [hunk 1]
```diff
@@ -112,7 +120,27 @@ $ git commit -m 'Update vue-strap version to v2.0.1-markbind.XYZ'
 
 * **Note:** We uniquely do this for each MarkBind release (rather than spontaneously update the vue-strap files for each affected PR), in order to reduce unnecessary merge conflicts. It also makes it easier for the maintainers to vet the changes.
 
-2. Rebuild the test files.
+2. Do an `npm version` to increment the version number. Which to increment (`patch`, `minor` or `major`) depends on what PRs are merged for the new version, which means you must know beforehand about the changes.
```
Comment (Chng-Zhi-Xuan) [inline docs/devGuide/maintainerGuide.md]: Do ~an~ -> a `npm version` ...

File: docs/devGuide/maintainerGuide.md [hunk 2]
```diff
@@ -142,48 +170,33 @@ index 779f279..bb3c602 100644
...
-4. Push the new commit and version tag generated by `npm` (change `vA.B.C` to the new version's string accordingly).
+5. Push the new commit and version tag generated by `npm` (change `vA.B.C` to the new version's string accordingly).
 
 ```
 $ git push upstream master
 $ git push upstream vA.B.C
 ```
 
-5. Do an `npm publish`. You should receive a notification by `npm` that the publish is successful.
+6. Do an `npm publish`. You should receive a notification by `npm` that the publish is successful.
```
Comment (Chng-Zhi-Xuan) [inline docs/devGuide/maintainerGuide.md]: Do ~an~ a `npm publish` ...

File: docs/devGuide/maintainerGuide.md [hunk 3]
```diff
@@ -142,48 +170,33 @@ index 779f279..bb3c602 100644
...
+7. If you used `npm link` for your project, ensure that you try the new release on another platform that doesn't use `npm link`, so that we can be sure the end-users can install and use the new version.
 
     a. Just do `npm i -g markbind-cli@A.B.C` on the different platform and...
 
     b. ... play around with the new MarkBind version to ensure that it works.
     
-7. Ensure that the documentation at markbind.org is deployed correctly.
+8. Ensure that the documentation at markbind.org is deployed correctly.
 
 Since #701, the deployment is automated.
```
Comment (Chng-Zhi-Xuan) [inline docs/devGuide/maintainerGuide.md]: Should we hyperlink `#701`?

File: docs/devGuide/maintainerGuide.md (hunk 1 above)
Comment (acjh) [reply docs/devGuide/maintainerGuide.md]: Actually, "an npm version" is correct. As with "a user".
https://ell.stackexchange.com/questions/26986/why-a-user-instead-of-an-user

**Comment by yamgent** [comment]
Updated.

**Pull request merged**